.github/workflows/validation_workflow.yml

```diff
@@ -13,14 +13,21 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: [ "3.10", "3.8","3.9"]
```

Comment:
this change looks unnecessary

Reply:
ah right, just due to the multiple changes. but I guess still fine to keep this